style/base.css

```diff
@@ -148,3 +148,64 @@
   background: var(--jp-layout-color2);
   flex-direction: row-reverse;
 }
+
+.glue-LinkEditor-widget {
+  border: solid 2px var(--jp-border-color1);
+}
+
+.glue-LinkEditor-title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--jp-layout-color4);
+  min-height: 25px;
+  color: var(--jp-ui-inverse-font-color0);
+  font-weight: bold;
+}
+
+.glue-LinkEditor-tabToolbar {
+  justify-content: space-around;
+}
+
+.glue-LinkEditor-tabToolbar .selected {
+  text-decoration: underline;
+  color: var(--jp-content-link-color);
```

Comment:
The tabs really looks like hyperlinks with this CSS. We could maybe improve this look to look more like tabs. This could definitely be done in a separate PR though.